protocol/messenger.go

```diff
@@ -74,12 +74,12 @@ type RawResponse struct {
 }
 
 type MessengerResponse struct {
-	Chats         []*Chat                     `json:"chats,omitEmpty"`
-	Messages      []*Message                  `json:"messages,omitEmpty"`
-	Contacts      []*Contact                  `json:"contacts,omitEmpty"`
-	Installations []*multidevice.Installation `json:"installations,omitEmpty"`
+	Chats         []*Chat                     `json:"chats,omitempty"`
```

Comment:
Potentially breaking change cc @cammellos.

Reply:
This is also fine